docs/api/structures/display.md

```diff
@@ -10,6 +10,7 @@
 * `colorSpace` String -  represent a color space (three-dimensional object which contains all realizable color combinations) for the purpose of color conversions
 * `colorDepth` Number - The number of bits per pixel.
 * `depthPerComponent` Number - The number of bits per color component.
+* `displayFrequency` Number _Windows_ - The display refresh rate.
```

Comment:
It looks like this might be supported on Linux, and possibly on Mac as well? 
https://source.chromium.org/chromium/chromium/src/+/master:ui/base/x/x11_display_util.cc;l=313;drc=3980dd6e559b6d986dd29439ebbe5c9e6b7cb700
https://source.chromium.org/chromium/chromium/src/+/master:ui/display/mac/screen_mac.mm;l=139;drc=d29aea5fe2e3cafa6a84fda198d2e4c4ac5b1832

Reply:
oh looks like you're right! the OG CL said windows-only but it seems they've updated since then.